output-tests/draft2019-09/output-schema.json

```diff
@@ -3,7 +3,7 @@
   "$id": "https://json-schema.org/draft/2019-09/output/schema",
   "description": "A schema that validates the minimum requirements for validation output",
 
-  "anyOf": [
+  "oneOf": [
```

Comment:
This needs to be `anyOf` because `flag` is always valid since it's only the `valid` property.  `oneOf` will fail.